srcpkgs/nim/template

```diff
@@ -1,12 +1,12 @@
 # Template file for 'nim'
 pkgname=nim
-version=1.2.6
-revision=2
+version=1.4.0
+revision=1
 _cversion=0.20.0
-_nimbleversion=0.11.4
+_nimbleversion=0.12.0
 create_wrksrc=yes
 build_wrksrc="Nim-$version"
-hostmakedepends="ed"
+hostmakedepends="ed git"
```

Comment:
What is this needed for

Reply:
the bootstrap script downloads the "fusion" package ( https://github.com/nim-lang/fusion )

technically nimble also soft-requires git, being a package manager that almost entirely works through github in practice.